platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/AbstractPKIHandler.java

```diff
@@ -103,7 +103,14 @@ public HandlerResult getNormalizedToken(
     } else {
       String errorMsg = "The certificate used to complete the request has been revoked.";
       LOGGER.info(errorMsg);
-      throw new ServletException(errorMsg);
+      try {
+        httpResponse.sendError(
+            HttpServletResponse.SC_FORBIDDEN, "Your certificate has been revoked.");
```

Comment:
"has been revoked" makes it sound like DIB just revoked their certificate. 

How about something more passive like:
"Client certificate is invalid.  The certificate or one of its Certificate Authorities is revoked."